src/style.css

```
@@ -0,0 +1,5 @@
+* {
+    margin: 0;
+    padding: 0;

```

Ficaria mais bonito modificar o margin e paddin para 20px

```suggestion
    margin: 20px;
    padding: 20px;

```

---

o que voce acha de modificar para 10px?